front-end/linux-help.html

```diff
@@ -42,9 +42,10 @@
             </p>
             <p class="code">
                 sudo usermod -a -G dialout $USER
+		sg dialout "npm start"
```

Comment:
When you download the binaries "npm start" won't be the executable. What does this command do exactly?